frontend/src/App.vue

```diff
@@ -59,8 +59,12 @@ const paths = [
     label: 'Maps'
   },
   {
-    attrs: { to: '/resources' },
-    label: 'Resources'
+    attrs: { to: '/floodsavvy' },
+    label: 'FloodSavvy Resources'
+  },
+  {
+    attrs: { to: '/nwps' },
+    label: 'NWPS Resources'
```

Comment:
I think this would look a lot better with a dropdown select "Resources" -> NWPS + FS